### ISSUE TITLE
Error on conditional child component on AccordionItem

### ISSUE THREAD

**Issue by janzenz** [springload/react-accessible-accordion#21]
When adding a conditional component under `<AccordionItem/>` component I'm getting this error:
```console
Warning: Failed prop type: Invalid prop `children[2]` of type `boolean` supplied to `AccordionItem`, expected a single ReactElement.
```

### EDIT
@vincentaudebert sorry my description was hurried yesterday. Basically what I'm trying to do was add a **conditional** component inside the `<AccordionItem>` component. However when I try to do so I get an error as per the Warning message above.

So here's my use-case, I wanted to add a **conditional** child component, for example a `<div/>` tag inside the `<AccordionItem>`, like so:
```jsx
<AccordionItem>
  {showDiv === true && <div/>}
</AccordionItem>
```

I believe the problem is because this statement `{showDiv === true && <div/>}` returns a boolean if `showDiv === true` is **falsy**. I could try the following:
```jsx
{showDiv === true ? <div/> : null}
```
However, this throws error as well as `null` isn't the expected type of `<AccordionItem/>` so our workaround in the **ACC** project is to enclose everything inside a parent component.
```jsx
<AccordionItem>
  <div className="parent-component">
    {showDiv === true && <div />}
  </div>
</AccordionItem>
```

I think the fix for this is to allow a **mixed** child type for the **AccordionItem** component. But I'm not sure if it's a thing this library is keen to support.

**Comment by ryami333** [comment]
For what it's worth, this would be solved by https://github.com/springload/react-accessible-accordion/issues/20.

Otherwise, I don't see any issue with allowing a mixed child type. IMO, this library should allow for more bespoke layout options, and that's why I've started work on another branch which aims to allow for this.

**Comment by vincentaudebert** [comment]
Sorry guys I'm planning on reviewing and merging Flow PR ASAP as indeed it will solve this issue.
I've just been dragged a bit on not open source work :)

I'll let you know when it's merged and will probably make it `0.7.0`

**Comment by vincentaudebert** [comment]
Should be all good now with `1.0.0` let me know @janzenz
